network/src/test/java/bisq/network/p2p/services/peergroup/exchange/BasePeerExchangeServiceTest.java

```diff
@@ -41,24 +44,32 @@
     void test_peerExchange(Node.Config nodeConfig) throws InterruptedException, ExecutionException {
         int numSeeds = 2;
         int numNodes = 2;
-        PeerGroupStore peerGroupStore = new PeerGroupStore();
         BanList banList = new BanList();
+        Node tempNode = new Node(banList, nodeConfig, "node-id");
+        PeerGroupStore peerGroupStore = new PeerGroupStore();
+        PersistenceService persistenceService = new PersistenceService(getBaseDirName());
```

Comment:
For really testing persistence I guess it requires more to setup the persistence framework.

Reply:
Yes, that was only to make the test compile after the constructor change.

Haven't found a way yet to fully test persistence, although some tests worked (like `ClearNetPeerExchangeServiceTest` when enabled).